src/matcha_ml/cli/ui/functions.py

```diff
@@ -0,0 +1,32 @@
+"""UI functions."""
+from typing import List, Tuple
+
+import typer
+
+from matcha_ml.cli.ui.emojis import Emojis
+from matcha_ml.cli.ui.print_messages import (
+    print_status,
+)
+from matcha_ml.cli.ui.status_message_builders import (
+    build_resource_confirmation,
+)
+
+
+def get_modify_resource_approval(verb: str, resources: List[Tuple[str, str]]) -> bool:
```

Comment:
For functions that return a bool, this function name is slightly more descriptive: `is_user_approved()`

Reply:
Updated, thanks Jon